components/HomeHero/HomeHero.tsx

```diff
@@ -0,0 +1,28 @@
+import Image from 'next/image';
+
+import logo from '../../public/images/headless-logo.svg';
+
+import styles from './HomeHero.module.scss';
+
+import Strapline from './Strapline';
+import type { StraplineProps } from './Strapline/types';
+
+const HomeHero = ({ icon, text }: StraplineProps) => (
```

Comment:
```suggestion
const Hero = ({ icon, text }: HeroProps) => (
```
If we're making the type generic across components, I'd set it to the parent to avoid confusion (it's more common for child to be a subset of the parent, rather than the parent to be a subset of the child)

I'd also refactor the file structure and rename the component to `Hero`. If we add a `Homepage` directory within the components directory, we could add this component as a subfolder and name it `Hero` without conflicts.

Reply:
> I'd also refactor the file structure and rename the component to Hero. If we add a Homepage directory within the components directory, we could add this component as a subfolder and name it Hero without conflicts.

That would make us add a directory that'll only contain one subfolder. If we keep it as is, we can add `TopicPageHero` component within the components directory without having to add another `TopicPage` folder where we'll add a `Hero` component.

Reply:
I would consider the possibility that more custom components will be added that are just relevant to one page, making the added folder worthwhile. I am biased here as this is what I've experienced working on the careers site, but as this site doesn't have many components that are specific to one page, your suggestion should be fine / refactored later if necessary.

Reply:
I get your point. We can always refactor if needed indeed.